.github/workflows/release_flank_scripts.yml

```diff
@@ -25,9 +25,10 @@ jobs:
     - uses: actions/checkout@v4
     - uses: gradle/wrapper-validation-action@v3
 
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@v4
+
     - name: Gradle Upload to Github packages and Github release
-      uses: gradle/gradle-build-action@v3.5.0
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        arguments: "flank-scripts:releaseFlankScripts -PGITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}"
+      run: ./gradlew flank-scripts:releaseFlankScripts -PGITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
```

Comment:
You already have `$GITHUB_TOKEN` available from `env`. 

That said, best practice is to rename that specific `env` variable `GH_TOKEN`. GitHub doesn't want people defining variables in their `GITHUB_... ` "namespace".